cmd/umoci/config.go

```diff
@@ -120,6 +120,22 @@ func parseEnv(env string) (string, string, error) {
 	return name, value, nil
 }
 
+// parseLabel splits a given label (of the form name=value) into (name,
+// value). An error is returned if there is no "=" in the line or if the
+// name is empty.
+func parseLabel(label string) (string, string, error) {
```

Comment:
obviously a lot of redundancy with parseEnv, but... golang. lmk if you'd prefer a different approach

Reply:
Maybe just rename `parseEnv` to `parseKV` and then just make the error message generic (perhaps using `errors.Wrapf` in the caller)?